### ISSUE TITLE
BlueStyle: Trailing comma on single line

### ISSUE THREAD

**Issue by omus** [domluna/JuliaFormatter.jl#455]
I noticed the JuliaFormatter was removing the trailing comma in this example:
```julia
@test S3Path("s3://my_bucket/prefix/that/is/fun") == S3Path(
    ("prefix", "that", "is", "fun"), "/", "s3://my_bucket", false, nothing, cfg,
)
```
https://github.com/JuliaCloud/AWSS3.jl/pull/192#discussion_r689630860

I think the trailing comma is supposed to be included here while in this example it should not be:
```julia
@test S3Path("s3://my_bucket/prefix/that/is/fun") == 
    S3Path(("prefix", "that", "is", "fun"), "/", "s3://my_bucket", false, nothing, cfg)
```

**Comment by domluna** [comment]
With BlueStyle when the arguments can be fit on 1 line (on the nested line), i.e.

```
foo(
    arg1, arg2, arg3
)
```

There is no trailing comma

I don't think there's a convention on keeping the trailing comma when the right parenthesis directly follows. It is also removed in DefaultStyle

cc @nickrobinson251

**Comment by nickrobinson251** [comment]
Yeah I think the current behaviour (no trailing comma on single line) is the agreed upon one for BlueStyle

 (I don't have the discussion to hand, but I _think_ the current behaviour was something @iamed2 requested, maybe. Could have been someone else.)

The trailing comma on the single line doesn't effect the diff when another arg is added, unlike a trailing comma on arguments split over multiple lines where a trailing comma allows adding a new arg without having to change 2 lines. Also a trailer comma in the multi-line case makes rearranging args easier, which also doesn't apply to single-line case. Finally, i think it felt particularly unnecessary when adding a trailing comma to the single-line case takes the line past the char limit and causes the args to need to then be split over multiple lines. (I think these were the reasons stated for the current policy)

Also in such a minor csse, I think BlueStyle being the same as DefaultStyle is quite nice (but that's less important than the point above I this)

**Comment by omus** [comment]
I'll open an issue in the BlueStyle repo. It's not explicitly called out but there are a few examples in the style of:
```julia
arr = [
    1, 2, 3,
]
```

**Comment by penelopeysm** [comment]
In the absence of a resolution to https://github.com/JuliaDiff/BlueStyle/issues/100, I think it's fair to close this as intended on the formatter side.
